build/NuGetPackages/Microsoft.Build.Tasks.Core.nuspec

```diff
@@ -67,6 +67,7 @@
         <dependency id="Microsoft.Build.Utilities.Core" version="[$version$]" />
         <dependency id="System.Collections.Immutable" version="1.3.1" />
         <dependency id="System.Reflection.Metadata" version="1.3.0" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" />
```

Comment:
Oh, I see: `System.Runtime.InteropServices.OSPlatform` only became available for netstandard in `4.0.2.0`. https://apisof.net/catalog/System.Runtime.InteropServices.OSPlatform. And the only package newer than that is `4.3.0`. Well, that's . . . something.